actor/src/main/scala-2.12/org/apache/pekko/util/ByteString.scala

```diff
@@ -827,6 +827,14 @@ sealed abstract class ByteString extends IndexedSeq[Byte] with IndexedSeqOptimiz
    */
   def toArrayUnsafe(): Array[Byte] = toArray
 
+  /**
+   * Return the bytes in this ByteString as an InputStream.
+   *
+   * @return the bytes in this ByteString accessible as an InputStream
+   * @since 1.1.0
+   */
+  def getInputStream(): InputStream = new ByteArrayInputStream(toArrayUnsafe())
```

Comment:
For composiable Bytestring, I expect this call lazy.

Reply:
How about name it `asInputString` or `toInputStream`?

Reply:
renamed as `asInputStream` because it is similar to `asByteBuffer`